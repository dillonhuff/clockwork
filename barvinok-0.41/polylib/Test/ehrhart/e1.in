# This is the first example in the Ehrhart paper
4 6
1  1  0 0 0 0	i >= 0
1 -2  0 1 0 0	-2i + P >= 0
1  0  1 0 0 0	j >= 0
1  0 -2 0 1 0	-2j + Q >= 0

2 4
1 1 0 0		P >= 0
1 0 1 0		Q >= 0

P Q
CL'96 example
