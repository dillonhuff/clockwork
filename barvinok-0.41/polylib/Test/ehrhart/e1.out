---------------------------------------
Domain:
         P  >= 0
         Q  >= 0
          1 >= 0

Vertices:
[ P/2,  Q/2 ]
[ P/2,  0 ]
[ 0,  Q/2 ]
[ 0,  0 ]

Ehrhart Polynomial:
( ( 1/4 * Q + [ 1/2, 1/4 ]_Q )
 * P + [ ( 1/2 * Q + [ 1, 1/2 ]_Q )
, ( 1/4 * Q + [ 1/2, 1/4 ]_Q )
 ]_P )

